frontend/packages/ceph-storage-plugin/integration-tests/tests/installFlow.scenario.ts

```diff
@@ -0,0 +1,254 @@
+import { execSync } from 'child_process';
+import { ExpectedConditions as until, browser } from 'protractor';
+import * as crudView from '@console/internal-integration-tests/views/crud.view';
+import {
+  goToOperatorHub,
+  installOperator,
+  InstallCluster,
+  searchInOperatorHub,
+  uninstallOperator,
+  closeOperatorInstall,
+  storageClusterLink,
+  btnCreate,
+  selectWorkerRows,
+  status,
+  NS,
+  operatorsPage,
+  ocsOperator,
+  ocsOperatorStatus,
+  filterInput,
+  getDataFromRowAndCol,
+  podNameFilter,
+  goToStorageClasses,
+  checkIfClusterIsReady,
+  getPodData,
+  getPodPhase,
+  getPodName,
+} from '../views/subscriptionFlow.view';
+
+const installer = new InstallCluster(NS);
+
+const ocsDeploy = `
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    openshift.io/cluster-monitoring: "true"
+  name: openshift-storage
+spec: {}
+---
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: openshift-storage-operatorgroup
+  namespace: openshift-storage
+spec:
+  serviceAccount:
+    metadata:
+      creationTimestamp: null
+  targetNamespaces:
+  - openshift-storage
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: ocs-catalogsource
+  namespace: openshift-marketplace
+spec:
+  displayName: Openshift Container Storage
+  icon:
+    base64data: ""
+    mediatype: ""
+  image: quay.io/ocs-dev/ocs-registry:latest
+  publisher: Red Hat
+  sourceType: grpc
+---`;
+
+describe('Testing OCS Subscription', () => {
+  beforeAll(async () => {
+    await execSync(`echo '${ocsDeploy}' | oc apply -f -`);
+    // Let the cluster add defns
+    await browser.sleep(45000);
+  }, 100000);
+
+  it('Subscribe to Operator', async () => {
+    await browser.wait(until.and(crudView.untilNoLoadersPresent));
+    await goToOperatorHub();
+    await browser.wait(until.and(crudView.untilNoLoadersPresent));
+    const ocs = await searchInOperatorHub('Openshift Container Storage');
+    ocs.click();
+    let uninstall = null;
+    try {
+      uninstall = !!(await uninstallOperator.isDisplayed());
+    } catch {
+      uninstall = false;
+    }
+    if (uninstall) {
+      // eslint-disable-next-line
+      console.log('Already subscribed to the operator');
+      await browser.wait(until.visibilityOf(closeOperatorInstall));
+      await closeOperatorInstall.click();
+      await operatorsPage();
+    } else {
+      await browser.wait(until.visibilityOf(installOperator));
+      await installOperator.click();
+      browser.refresh();
+      await installer.installOperator();
+    }
+    await browser.wait(until.visibilityOf(ocsOperator));
+    expect(ocsOperator.isDisplayed()).toBe(true);
+    await browser.wait(until.visibilityOf(ocsOperatorStatus));
+    let text = await ocsOperatorStatus.getText();
+    // If not installed yet wait for 15 seconds
+    if (!text.includes('Succeeded')) await browser.sleep(10000);
```

Comment:
Create a separate loop for this with smaller sleep times.? @rhrazdil what do you think?